src/track/track_gps_l1ca.c

```diff
@@ -337,11 +344,20 @@ static void tracker_gps_l1ca_update(const tracker_channel_info_t *channel_info,
 
     /* Indicate that a mode change has occurred. */
     common_data->mode_change_count = common_data->update_count;
+
+    do_l1ca_to_l2cm_handover(common_data->sample_count,
```

Comment:
What happens if the L2CM tracking channel fails to start or is dropped? It should probably attempt handover periodically, though I'm not sure the best way to accomplish that.